install/solutions-hub/dashboards/install.yml

```diff
@@ -0,0 +1,14 @@
+id: solutions-hub-dashboards
+name: solutions-hub
+title: New Relic Solutions Hub Dashboards
+description: New Relic's solution framework offers a repeatable and proven approach to help you do just that. Using proven best practices and expert guidance, your organization can move faster, stay on course, and transform successfully.
+  
+
+target:
```

Comment:
I'm not sure if the unknown value will cause an issue here. Was this what you were instructed to do here @JimHagan ?

Reply:
We confirmed with @jbeveland27 what will occur in the UI in this case, and though it's not ideal it did identify a potential improvement for us to look at.